src/backend/api/handlers/helpers/model_properties.py

```diff
@@ -44,6 +44,16 @@
     "predicted_time",
 ]
 
+search_team_properties = [
+    "key",
+    "nickname",
+]
+
+search_event_properties = [
+    "key",
+    "name",
```

Comment:
on prod, we use short name. We have two options:
1) Include both name and short name and choose on the client side which to use (don't love this idea because it bloats this model)
2) Make `short_name` not nullable in the API (defaulting to `name` if it's null) and use that here.

Reply:
Are you sure prod uses short_name? Here's 2024 Chezy (abbreviated):

```json
{
  "address": "960 W Hedding St, San Jose, CA 95126, USA",
  "city": "San Jose",
  "country": "USA",
  "district": null,
  "division_keys": [],
  "end_date": "2024-09-29",
  "event_code": "cc",
  "event_type": 99,
  "event_type_string": "Offseason",
  "first_event_code": null,
  "first_event_id": null,
  "key": "2024cc",
  "location_name": "Bellarmine College Preparatory",
  "name": "Chezy Champs",
  "short_name": "",
}
```

Reply:
Prod uses short name but defaults to name. Look at any of the divisions "sponsored by X"